lib/services/schema-object-factory.ts

```diff
@@ -424,15 +425,20 @@ export class SchemaObjectFactory {
     type: string | Record<string, any>
   ): SchemaObjectMetadata {
     const keysToRemove = ['type', 'enum'];
+    const keysToMove = [
+      'format',
```

Comment:
The original bug report was only for `format`, but these other keys are all similarly applicable to the inner type and are fields that my own projects use. There may be other fields that belong here.